redbot/cogs/mutes/mutes.py

```diff
@@ -1546,6 +1546,9 @@ async def mute_user(
             if not guild.me.guild_permissions.manage_roles or role >= guild.me.top_role:
                 ret["reason"] = _(MUTE_UNMUTE_ISSUES["permissions_issue_role"])
                 return ret
+            if role in user.roles:
```

Comment:
As Jack stated in #5161 we should care more about `[p]unmute` not so much `[p]mute`. This would prevent updating a mute duration forcing the user to unmute and then re-apply the mute which is inconvenient.

Reply:
you make a good point.

add_roles won't fail if the user already has the role and since the user is already muted the user gets what they expected.

It might be worthwhile to add a message for updating the mute duration and printing out already muted if none is given, but this is not the point of this PR

Reply:
Here is something along the lines of what I mean (keeping the already existing text)
![image](https://user-images.githubusercontent.com/15076013/146062879-a32597c3-515b-456f-aa5d-7e72417f9465.png)